### DISCUSSION TITLE
Picture is clear OCR is out of track with crazy guess

### DISCUSSION THREAD

**Discussion by jackoncept** [jomjol/AI-on-the-edge-device#3375] [Q&A]
Hi all, I am fighting with the model to get a proper reading, I ha ve tried digclass11 et digclass100 and I have some really random readings, any hints ?

<img width="697" alt="Capture d’écran 2024-11-07 à 21 03 24" src="https://github.com/user-attachments/assets/db8b8413-a434-4b91-8a8c-e2f7f7052f4d">

**Comment by SybexX** [comment]
![384122866-db8b8413-a434-4b91-8a8c-e2f7f7052f4d](https://github.com/user-attachments/assets/7408cf4d-5961-4a7f-b138-24a0b59e35f7)
